cadence/nfts/common/transactions/destroy_nfts.template.cdc

```diff
@@ -0,0 +1,30 @@
+import NonFungibleToken from {{{ imports.NonFungibleToken }}}
+import {{ contractName }} from {{{ contractAddress }}}
+
+/// This transaction withdraws multiple NFTs from the signer's collection and destroys them.
+///
+transaction(ids: [UInt64], fromBucketName: String?) {
+
+    /// A reference to the signer's {{ contractName }} collection.
+    ///
+    let collectionRef: &{{ contractName }}.Collection
+
+    prepare(signer: AuthAccount) {
+        
+        // Derive the collection path from the bucket name
+        let collectionName = {{ contractName }}.makeCollectionName(bucketName: fromBucketName)
+        let collectionStoragePath = {{ contractName }}.getStoragePath(suffix: collectionName)
+
+        self.collectionRef = signer.borrow<&{{ contractName }}.Collection>(from: collectionStoragePath)
+            ?? panic("failed to borrow collection")
+    }
+
+    execute {
+        for id in ids {
+            // withdraw the NFT from the signers's collection
+            let nft <- self.collectionRef.withdraw(withdrawID: id)
+
+            destroy nft
+        }
+    }
+}
```

Comment:
Small nitpick, but this file is missing a newline character. I like to end each file with an empty line so that it's compatible with legacy systems.